### DISCUSSION TITLE
composable rememberMarker to views

### DISCUSSION THREAD

**Discussion by Kotletka89** [patrykandpatrick/vico#728] [Questions]
How to make composable rememberMarker function from sample app work on views?

**Comment by Gowsky** [comment]
Hello! Every Compose API in Vico has its non-Compose counterpart. To find needed APIs, you can check composable functions' source code, the API reference, or [the wiki](https://patrykandpatrick.com/vico/wiki/components).

**Reply by Kotletka89** [reply]
Still don't know how to make same marker as remember marker 

https://github.com/patrykandpatrick/vico/blob/master/sample/src/main/java/com/patrykandpatrick/vico/sample/showcase/Marker.kt

**Reply by Gowsky** [reply]
Can you explain what **exactly** you do not know? Here's an example of what to do with `ShapeComponent`.
```diff
- rememberShapeComponent(labelBackgroundShape, surfaceColor)
-       .setShadow(radius = 4f, dy = 2f, applyElevationOverlay = true)
+ ShapeComponent(labelBackgroundShape, surfaceColor)
+     .setShadow(radius = 4f, dy = 2f, applyElevationOverlay = true)
```
